Makefile.PL

```diff
@@ -24,6 +24,8 @@ my %WriteMakefileArgs = (
         'Syntax::Keyword::Try'   => '0.25',
         'Template::Tiny::Strict' => '1.18',
         'Regexp::Common'         => '2017060201',
+        'Readonly'               => '2.05',
```

Comment:
The `Readonly` was left in there by accident. I've removed it and pushed. Can you remove this line and then I'll approve?